packages/block-editor/src/components/index.native.js

```diff
@@ -19,7 +19,6 @@ export * from './gradients';
 export * from './font-sizes';
 export { AlignmentControl, AlignmentToolbar } from './alignment-control';
 export { default as InnerBlocks } from './inner-blocks';
-export { default as InspectorAdvancedControls } from './inspector-advanced-controls';
```

Comment:
I hope it wasn't used in React Native so far. I couldn't locate any reference in the code.